### PR TITLE
studio/start: faster to value demo

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Follow-up to https://github.com/iterative/dvc.org/pull/4069#pullrequestreview-1161157744

### CODE REVIEW

File: content/docs/studio/get-started.md [hunk 1]
```diff
@@ -5,16 +5,10 @@ Studio].
 
 <admon>
 
-Iterative Studio [works very closely with your Git ecosystem][how-it-works]. So,
-you need to connect Iterative Studio to the Git repositories for your projects.
```
Comment (jorgeorpinel) [inline content/docs/studio/get-started.md:9]: > > added a brief about how Studio works in the get started page
>
> This should be covered in the Get Started itself, I think

And it is, in the **Sign up / sign in** section below, right?

image

We could update that section and/or link to https://dvc.org/doc/studio/how-it-works from there but TBH I'm not seeing much info on connecting Gut repos there. Maybe to https://dvc.org/doc/studio/user-guide/prepare-your-repositories ?

File: content/docs/studio/get-started.md [hunk 2]
```diff
@@ -5,16 +5,10 @@ Studio].
...
+We have renamed DVC Studio to Iterative Studio.
 
 </admon>
 
-> Note that we have renamed DVC Studio to Iterative Studio and Views to
-> Projects.
-
-https://www.youtube.com/watch?v=hKf4twg832g
-
```
Comment (tapadipti) [inline content/docs/studio/get-started.md:17]: Why is the video removed? And the message about renaming is meant to accompany the video, which is why it's styled as a short note, not admon.

File: content/docs/studio/index.md [hunk 3]
```diff
@@ -7,16 +7,18 @@ tools [DVC](https://dvc.org/), [CML](https://cml.dev), [MLEM](https://mlem.ai/),
 enables you to seamlessly manage data and machine learning models, run and track
 experiments, and visualize and share results.
 
-<cards>
+https://www.youtube.com/watch?v=hKf4twg832g
```
Comment (tapadipti) [inline content/docs/studio/index.md:10]: If we move the video to this page, we should also move this note: `Note that we have renamed DVC Studio to Iterative Studio and Views to Projects.` Coz the video uses the outdated terms.

File: content/docs/studio/get-started.md (hunk 1 above)
Comment (tapadipti) [reply content/docs/studio/get-started.md:9]: I can try to make that change.

File: content/docs/studio/index.md (hunk 3 above)
Comment (tapadipti) [reply content/docs/studio/index.md:13]: Updated.

File: content/docs/studio/get-started.md (hunk 2 above)
Comment (tapadipti) [reply content/docs/studio/get-started.md:17]: Updated.

File: content/docs/studio/get-started.md (hunk 1 above)
Comment (tapadipti) [reply content/docs/studio/get-started.md:9]: Updated.

**Review by tapadipti — APPROVED** [review]
lgtm. Thanks @jorgeorpinel

File: content/docs/studio/get-started.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/studio/get-started.md:9]: Thanks for the updates @tapadipti . I've made the intro even shorter now.

Comment (jorgeorpinel) [reply content/docs/studio/get-started.md:9]: PTAL

File: content/docs/studio/index.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/studio/index.md:13]: Sounds good. We should always use `<admon>` though, I think (updating).

**Review by tapadipti — APPROVED** [review]
@jorgeorpinel I've added [some minor changes](https://github.com/iterative/dvc.org/pull/4104/commits/17ad49ce9415986496dbb042ea1128e47cf13002). Thanks for the work!

**Comment by jorgeorpinel** [comment]
Thanks @tapadipti

**Pull request merged**